### PR TITLE
[Bug#96774] [BIC Common] Jasper Reports encrypted PDF creation fails

### PR DESCRIPTION
**Pull request by ivaklinov**
This pull request contains the proposed Seeburger changes on top of the JasperSoft JS7 original sources.

### CODE REVIEW

File: META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Manifest-Version: 1.0
 Ant-Version: Apache Ant 1.9.6
-Created-By: 1.8.0_191-b12 (Oracle Corporation)
+Created-By: 1.8.0_91-b14 (Oracle Corporation)
```
Comment (ecki) [inline META-INF/MANIFEST.MF]: hm, i would keep the original file or .gitignore it if it is generated. but downgrading to such an old version looks strange.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                            <configuration>
+                                <failOnError>false</failOnError>
+                            </configuration>
+                        </execution>
+                    </executions>
+            </plugin>
+		</plugins>
+	</build>
+
+   <!-- SEEBURGER repository changes -->
```
Comment (ecki) [inline pom.xml]: can we avoid that, it should be possible to overwrite that in jenkins. would like to have source which works for general public.

File: .classpath [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="src" path="core"/>
+	<classpathentry kind="src" path="rtf"/>
+	<classpathentry kind="src" path="rups"/>
+	<classpathentry kind="src" path="toolbox"/>
+	<classpathentry kind="lib" path="C:/Users/g.pazhev/.m2/repository/org/bouncycastle/bcprov-jdk15on/1.68/bcprov-jdk15on-1.68.jar"/>
```
Comment (ecki) [inline .classpath]: can you switch the project to maven container or at least not commit the .project file to avoid internal file names.

**Review by ecki — CHANGES_REQUESTED** [review]

File: META-INF/MANIFEST.MF (hunk 1 above)
Comment (ivaklinov) [reply META-INF/MANIFEST.MF]: OK

File: pom.xml (hunk 2 above)
Comment (ivaklinov) [reply pom.xml]: I do not think we can avoid this.... at least I do not know how.

File: .classpath (hunk 3 above)
Comment (ivaklinov) [reply .classpath]: OK. I will not commit the class-path file.

**Comment by ivaklinov** [comment]
The BC 1.68 fix is the DER -> ASN1 change. I will reinitialize the see-burger branch.
And make separate commits ... also address your other requested changes.

**Comment by ivaklinov** [comment]
I recreated the seeburger branch, Then I created two new commits only:

https://github.com/seeburger-ag/itext/commit/dd240f24aae69a7ef65b2ebf7838f3c6ff28c445
Seeburger changes for compatibility with BouncyCastle 1.68 

https://github.com/seeburger-ag/itext/commit/49e09f0edf51a4c4992108a72913b4f1da617cc4
Seeburger changes for the sake of DevOps.
Very minimal commit with a pom to build in Jenkins and remade updates.

**Pull request closed without merging**